en/tools/config-catalog-generator/data/apim.cache.gateway_token.toml

```diff
@@ -0,0 +1,3 @@
+[apim.cache.gateway_token]
+enable = true
+expiry_time = "900s" 
```

Comment:
Add newline at the end of file

Reply:
Extra line was not added since this file is only used to automatically generate config-catalog.md